M365/MDO/MDOThreatPolicyChecker.ps1

```diff
@@ -0,0 +1,840 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+.SYNOPSIS
+ This script checks which Microsoft Defender for Office 365 and Exchange Online Protection threat policies cover a particular user, including anti-malware, anti-phishing, inbound and outbound anti-spam, as well as Safe Attachments and Safe Links policies in case these are licensed for your tenant. In addition, the script can check for threat policies that have inclusion and/or exclusion settings that may be redundant or confusing and lead to missed coverage of users or coverage by an unexpected threat policy.
```

Comment:
_The `SYNOPSIS` provides a brief overview or summary of what the script or function does. It's meant to give you a quick idea of the functionality without going into too much detail._It could be something like this:
```suggestion
Evaluates user coverage and potential redundancies in Microsoft Defender for Office 365 and Exchange Online Protection threat policies, including anti-malware, anti-phishing, and anti-spam policies, as well as Safe Attachments and Safe Links policies if licensed.
```

At the moment, the `SYNOPSIS` is the same as the `DESCRIPTION` which doesn't make sense.